constants/members.json

```diff
@@ -3429,6 +3441,76 @@
       "term": 1235,
       "teams": ["shw"],
       "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fdefault.png?alt=media&token=fe95cc90-ba2b-4c04-a808-0f903cc8b519"
+    },
+    {
+      "name": "Niharika Srivatsa",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["haven"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Niharika_Srivatsa.jpg?alt=media&token=38202775-87fa-4078-8c6a-2aa7fa7bd1c9"
+    },
+    {
+      "name": "Jesse Li",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["internal-tools"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Jesse_Li.jpg?alt=media&token=c4ffd1f3-ef13-4683-87b3-aa7c7ec8006a"
+    },
+    {
+      "name": "Maggie Lu",
+      "role": "vp engineering",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVPE_Maggie_Lu.JPG?alt=media&token=ebc4f915-0d39-4934-a3b5-fea3c55a31d3"
+    },
+    {
+      "name": "Annie Cai",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["internal-tools"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2Fdev_annie_cai.jpg?alt=media&token=63b6e3b5-35a7-4151-b7fd-7ebb11abfb84"
+    },
+    {
+      "name": "Maisha Thasin",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["infra"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2Fdev_maisha_thasin.jpeg?alt=media&token=b4ddba89-ccd2-4345-b136-5b80475ca4a7"
+    },
+    {
+      "name": "Maisha Thasin",
```

Comment:
Guessing this should be for Shahan, but name is still Maisha

Reply:
@HeetShah can you help confirm?

Reply:
Looks like Maisha Thasin is listed as a developer for infra team. @LucyWuu this looks correct. @Jess2772 Shahan is technical lead for FCK. Did you mean someone/something else?

Reply:
Yea that one is fine. I was talking about lines 3474 and 3482, as they both have the same name field